pkg/operator/ceph/controller/predicate.go

```diff
@@ -94,7 +94,22 @@ func WatchControllerPredicate() predicate.Funcs {
 					return true
 				}
 
+			case *cephv1.CephNFS:
+				objNew := e.ObjectNew.(*cephv1.CephNFS)
+				logger.Debug("update event from the parent object CephNFS")
+				diff := cmp.Diff(objOld.Spec, objNew.Spec, resourceQtyComparer)
+				if diff != "" ||
```

Comment:
All these types have very similar template code. Can we factor any of it out to a helper method?

Reply:
Not really, since we don't know the type, and we don't want to reconstruct unstructured objects.